Samples/iOS-Swift/iOS-Swift/SentrySDKOverrides.swift

```diff
@@ -76,53 +72,82 @@ enum SentrySDKOverrides {
             }
         }
 
-        static var lifecycle: SentryProfileOptions.SentryProfileLifecycle {
+        /// - note: If no other overrides are present, we set the iOS-Swift app to use trace lifecycle (the SDK default is manual)
+        static var manualLifecycle: Bool {
             get {
-                getOverride(for: Profiling.Key.traceLifecycle) ? .trace : .manual
+                return getOverride(for: Profiling.Key.manualLifecycle)
             }
             set(newValue) {
-                UserDefaults.standard.set(newValue.rawValue, forKey: Profiling.Key.traceLifecycle.rawValue)
+                setOverride(for: Profiling.Key.manualLifecycle, value: newValue)
             }
         }
 
+        /// - note: If no other overrides are present, we set the iOS-Swift app to use launch profiling (the SDK default is to disable it)
         static var profileAppStarts: Bool {
             get {
-                getOverride(for: Profiling.Key.launchProfiling)
+                !getOverride(for: Profiling.Key.disableAppStartProfiling)
             }
             set(newValue) {
-                UserDefaults.standard.set(newValue, forKey: Profiling.Key.launchProfiling.rawValue)
+                setOverride(for: Profiling.Key.disableAppStartProfiling, value: !newValue)
             }
         }
+    }
```

Comment:
`l`: Please stay consistent in code style, either use explicit returns or not, both are fine (depending on our Swift lint setup)